testing/kuttl/scripts/pgbackrest-initialization.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+EXPECTED_STATUS=$1
+EXPECTED_NUM_BACKUPS=$2
+
+CLUSTER=${CLUSTER:-default}
+
+REPO_POD=$(kubectl -n "${NAMESPACE}" get pod --selector=postgres-operator.crunchydata.com/cluster="${CLUSTER}",postgres-operator.crunchydata.com/pgbackrest-dedicated --no-headers -o custom-columns=":metadata.name")
+
+INFO=$(kubectl -n "${NAMESPACE}" exec -i "${REPO_POD}" -c pgbackrest -- pgbackrest info)
```

Comment:
🌱 `kubectl exec` can do the pod lookup for us by naming a controller or service.

```suggestion
INFO=$(kubectl -n "${NAMESPACE}" exec "statefulset.apps/${CLUSTER}-repo-host" -c pgbackrest -- pgbackrest info)
```